public/app/app-init.js

```
@@ -0,0 +1,613 @@
+
+function wireModeButtons() {
+  // Live mode toggle (microphone)
+  document.getElementById('btn-live-mode')
+    ?.addEventListener('click', toggleLiveMode);
+
+  // Creator mode (process uploaded file with OfflineAudioContext)
+  document.getElementById('btn-creator-mode')
+    ?.addEventListener('click', () => processOffline('creator'));
+
+  // Forensic mode (ENTERPRISE only)
+  document.getElementById('btn-forensic-mode')
```

ESLint

[error] 'document' is not defined.

(no-undef)